### ISSUE TITLE
Does not parse multiple files

### ISSUE THREAD

**Issue by sqram** [erictsangx/koa-async-body#1]
When submitting multiple files via

```
var formData = new FormData()
formData.append('files[]', file, file.name);
```

it does not convert the files[] field into an array , and so files[]'s value will only hold information about the last appended file.

If i'm not doing anything wrong or missed something, I have a fix i'd like to submit

**Comment by erictsangx** [comment]
Yes, currently the module does not support multiple values with the same field name.
Feel free to submit a PR.
